pkgs/development/libraries/librttopo/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, geos
+}:
+
+stdenv.mkDerivation rec {
+  pname = "librttopo";
+  version = "1.1.0";
+
+  outputs = [ "out" "dev" ];
+
+  src = fetchFromGitHub {
+    githubBase = "git.osgeo.org/gitea";
```

Comment:
Does gitea work with fetchFromGitHub? Maybe we should use fetchGit instead.

Reply:
Yes, it's compatible.
I could add
```
fetchFromGitea = { domain, ... }@args: fetchFromGitHub (args // { githubBase = domain; })
```